controllers/constant/odlm.go

```diff
@@ -30,6 +30,7 @@ spec:
     spec:
       IBMLicensing:
         datasource: datacollector
+      IBMLicenseServiceReporter: {}
```

Comment:
Why we add them into the OperandConfig?

Reply:
I found it in size template, but now notice that we don't actually create this CR in the installation.

Will remove it.